server/avocano_api/cloudrun_helpers.py

```diff
@@ -73,11 +70,15 @@ def _service_name():
 
 def _service_url(project, region, service):
     try:
-        run_api = google_api("run", "v2")
+        client = run_v2.ServicesClient()
         fqname = f"projects/{project}/locations/{region}/services/{service}"
-        service = run_api.projects().locations().services().get(name=fqname).execute()
-        return service["uri"]
-    except (GAPIHTTPError, KeyError) as e:
+        request = run_v2.GetServiceRequest(name=fqname)
+        response = client.get_service(request=request)
+        return response.uri
+    except (
+        google.api_core.exceptions.PermissionDenied,
+        google.api_core.exceptions.NotFound,
```

Comment:
question: This looks odd to me. Are we ignoring/suppressing other errors?

Reply:
We should be ignoring all errors, yes; at best guess these were the errors that would appear. 

If there's a api_core.exceptions error that's a superset of errors that we can capture on, that'd be ideal

Reply:
Updated to use a common parent exception to capture all call errors